webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -12,9 +12,17 @@ class BreadcrumbBar extends React.Component {
       loadSwitcherResourcesByResource,
       currentPage,
       resourceUrl,
+      resourceSwitcherItems,
     } = this.props;
-
-    if (!currentPage || resourceUrl !== resource.resourceUrl) {
+    const checkUrlFormat = resourceSwitcherItems.length
```

Comment:
nitpick: `checkUrlFormat` would be a good name for a function while `isUrlFormatValid` would be a good name for a boolean variable.

Reply:
I agree, thanks